config.go

```diff
@@ -46,23 +51,28 @@ var (
 		GestureSupport:      false,
 		LoadBuiltinCommands: false,
 		TrackInputChanges:   false,
+		EventRefreshRate:    0 * time.Second,
+		DisableDebugConsole: false,
 	}
 )
 
 // Config stores initialization settings for oak.
 type Config struct {
-	Assets              Assets `json:"assets"`
-	Debug               Debug  `json:"debug"`
-	Screen              Screen `json:"screen"`
-	Font                Font   `json:"font"`
-	FrameRate           int    `json:"frameRate"`
-	DrawFrameRate       int    `json:"drawFrameRate"`
-	Language            string `json:"language"`
-	Title               string `json:"title"`
-	BatchLoad           bool   `json:"batchLoad"`
-	GestureSupport      bool   `json:"gestureSupport"`
-	LoadBuiltinCommands bool   `json:"loadBuiltinCommands"`
-	TrackInputChanges   bool   `json:"trackInputChanges"`
+	Assets              Assets           `json:"assets"`
+	Debug               Debug            `json:"debug"`
+	Screen              Screen           `json:"screen"`
+	Font                Font             `json:"font"`
+	BatchLoadOptions    BatchLoadOptions `json:"batchLoadOptions"`
+	FrameRate           int              `json:"frameRate"`
+	DrawFrameRate       int              `json:"drawFrameRate"`
+	Language            string           `json:"language"`
+	Title               string           `json:"title"`
+	EventRefreshRate    time.Duration    `json:"refreshRate"`
+	BatchLoad           bool             `json:"batchLoad"`
+	GestureSupport      bool             `json:"gestureSupport"`
+	LoadBuiltinCommands bool             `json:"loadBuiltinCommands"`
+	TrackInputChanges   bool             `json:"trackInputChanges"`
+	DisableDebugConsole bool             `json:"disableDebugConsole"`
```

Comment:
Not a note for this pull but a thought for oak 3.0.
Consider updating json naming to be non-camel but rather _ driven 
ie track_input_changes instead in the tag.

Not a big deal though.